### DISCUSSION TITLE
关于配置自签证书中转节点指定根证书的问题

### DISCUSSION THREAD

**Discussion by C726** [v2fly/v2ray-core#1252] [Q&A]
你好， 
我使用了自签TLS证书，用于中转服务器与海外服务器进行通讯，但是我未找到如何在程序中指定根证书

请问可以提供一下方法吗？

**Comment by BozhanL** [comment]
https://www.v2fly.org/config/transport.html#certificateobject
"usage": "verify"

**Reply by C726** [reply]
非常感谢！
